script.js

```diff
@@ -106,9 +168,7 @@ restaurantsApp.handleSearch = function () {
     let locationInput = $('#locationInput').val().trim(' ');
     let termInput = $('#termInput').val().trim(' ');
     //  Get user city and populate the locationTerm input
```

Comment:
could you delete this cm , I think I left it here

Reply:
Sure, removed.